src/CodeCompliance/Domain/Checks/NotUnique/Constant.php

```diff
@@ -54,4 +54,15 @@ public function getViolations(): array
 
         return $violations;
     }
+
+    /**
+     * @param string $value
+     * @param string $projectPrefix
+     *
+     * @return bool
+     */
+    protected function hasProjectPrefix(string $value, string $projectPrefix): bool
```

Comment:
I think you can reuse an existing method in the base class.

Reply:
Method was removed from current class